examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -263,6 +267,33 @@ export default class App extends PureComponent {
     return true;
   }
 
+  // TODO Find out what else we can show to user in debug mode.
+  prepareTileInfo(tileheader) {
```

Comment:
Maybe a function in a separate file `tile-debug.js`

```
function getTileDebugInfo(tileHeader)
```

Reply:
done